lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -117,15 +117,45 @@ export default function LaunchErrorDialog({
         </ErrorModal>
       );
 
-    case 'd2l_file_not_found_in_course':
+    case 'd2l_file_not_found_in_course_instructor':
       return (
         <ErrorModal
           {...defaultProps}
           title="Hypothesis couldn't find the file in the course"
         >
+          <p>This might have happened because:</p>
+          <ul className="px-4 list-disc">
+            <li>The file has been deleted from D2L</li>
+            <li>
+              The course was copied and the selected is not in the new course.
+            </li>
+          </ul>
+          <p>
+            To fix the issue recreate the assignment with a different file. Also
+            check our document about
+            <ExternalLink href="https://web.hypothes.is/help/using-hypothesis-with-d2l-course-content-files/">
+              Using Hypothesis With D2L Course Content Files
+            </ExternalLink>
+          </p>
```

Comment:
What does _recreate_ mean here, exactly? Is the implication that (without edit-assignment support) there is no way to fix the problem without creating a new assignment?

Note the explicit `{' '}` to ensure that there is a space between _about_ and the link. 

```suggestion
          <p>
            To fix the issue, recreate the assignment with a different file. More information can be found in our document about{' '}
            <ExternalLink href="https://web.hypothes.is/help/using-hypothesis-with-d2l-course-content-files/">
              Using Hypothesis With D2L Course Content Files
            </ExternalLink>.
          </p>
```

Reply:
Yeah, there's no way out of this other than to recreate the assignment in the LMS, the same is true for other LMS for both files and group issues.

The same wording/solution is issued in KB articles etc. Editing assignments will be a big deal for this issues.